### PR TITLE
Only allow tables at line start to fix arrays.

### PR DESCRIPTION
**Pull request by eerimoq**
Hi!

Arrays are matched as keywords on master. This PR aims to fix that.

On master:
image

With this PR:
image

### CODE REVIEW

File: pygments/lexers/configs.py [hunk 1]
```diff
@@ -968,9 +968,11 @@ class TOMLLexer(RegexLexer):
 
     tokens = {
         'root': [
+            # Table
+            (r'^(\s*)(\[.*?\])$', bygroups(Text, Keyword)),
 
             # Basics, comments, strings
-            (r'\s+', Text),
+            (r'[ \t]+', Text),
```
Comment (birkenfeld) [inline pygments/lexers/configs.py:975]: is this change necessary? You're not matching newlines between entries anywhere now; they are getting caught by the "out of rules" handler in lexer.py.

Comment (eerimoq) [reply pygments/lexers/configs.py:975]: Yes, otherwise tables with preceding spaces are not found. I guess `\s+` consumes the newline character.

image

image

Comment (birkenfeld) [reply pygments/lexers/configs.py:975]: Right, but then please include an explicit rule for `'\n'` too.

Comment (eerimoq) [reply pygments/lexers/configs.py:975]: Done.

Comment (eerimoq) [reply pygments/lexers/configs.py:975]: Anything else needed?

**Comment by birkenfeld** [comment]
LGTM, thanks!

**Comment by eerimoq** [comment]
When do you think the next release will be created?

**Comment by birkenfeld** [comment]
@Anteru will know :)

**Comment by Anteru** [comment]
I'm hoping for sometime during August. The milestone is tentatively scheduled for the first weekend in August, I hope to get around to cleaning up the release notes etc. and push for a release then.
